### ISSUE TITLE
Force client browser cache clear

### ISSUE THREAD

**Issue by Stono** [grafana/grafana#12472]
Hello!
We have an interesting deployment process for grafana and the dashboards through our CI/CD of numerous environments.

Sometimes we completely blow away Grafana (effectively treating it as stateless) and re-import the dashboards from `grafana.yaml`.

However, this causes some problems because it seems the folders are generated unique id's, and the clients browser is incorrectly caching the old folder.

Long story short; is there any way, or any flag i can set in our Grafana docker image that would forcefully reset the browser cache of the clients (like, re-keying it or similar).

Thanks

**Comment by torkelo** [comment]
Caching old folder? Not sure I understand. You go to the folder url in grafana and this change when you reprovision grafana ? There’s is no way to control the browser redirect cache, but for what urls is grafana redirecting you? This is will only happen if you use old dashboard urls without uid. The dashboard url will be the same after each clean provision if the json includes a uid.

Please provide more info on what urls your having problem with
